common/main/cpp/native_c4listener.cc

```diff
@@ -574,6 +574,8 @@ JNICALL Java_com_couchbase_lite_internal_core_impl_NativeC4Listener_startTls(
         jboolean enableDeltaSync,
         jboolean requirePasswordAuth) {
     C4TLSConfig tlsConfig = {};
+    tlsConfig.rootClientCerts = nullptr;
+    tlsConfig.certificate = nullptr;
```

Comment:
These two lines are redundant with the `{}` initializer (which basically memsets the whole thing to 0).

Reply:
Ah... I wondered about that.  I wasn't sure that was guaranteed to be initialized.  Will remove.